### DISCUSSION TITLE
linked table column

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#4889] [Ideas]
Hello,

Just upgraded to v1.0.82 and found a great enhancement on the table componemts, great works. Here I have a question, is there any plan to have a custom table column type feature, for example a link / icon / button table column? In my case, I have a location column, I want the column to be a link, anyone click this cell then will be link to the location page. 

![image](https://user-images.githubusercontent.com/29649651/157792111-87d8d177-6966-4036-a242-ba702eb8480e.png)

**Comment by mjashanks** [comment]
Hey Sam. We don't have any immediate plans for it, but this is a fantastic idea, thanks.

**Comment by stian90** [comment]
Just wanted to add that I also would love this future. This could add actions inside tables ass "task done" "download file" from s3 bucket and so on.

**Reply by sql4bucks** [reply]
You could accomplish this by letting us specify a column label when adding component columns. In the example above, if we could just label the second component column "Location" that would basically give the OP what he wanted. He could remove "location" as one of the selected columns to display and instead display it in a component column.

<img width="411" alt="image" src="https://user-images.githubusercontent.com/541000/201498601-94a09d20-d043-4467-94e0-d0604dc02a36.png">
